### DISCUSSION TITLE
Does the Application GPO Blacklist Policy prevent initial install or just updates?

### DISCUSSION THREAD

**Discussion by JasonCioffi** [Romanitho/Winget-AutoUpdate#602] [Q&A] [answered]
There are a few applications in our software roster, like Chrome and MS365, that I prefer to let update on their own.  Will the Application GPO Blacklist policy prevent installation of anything listed there?  Or does it just restrict updates to those listed?  Hopefully it's the latter.

**Comment by KnifMelti** [comment, marked as answer]
> There are a few applications in our software roster, like Chrome and MS365, that I prefer to let update on their own. Will the Application GPO Blacklist policy prevent installation of anything listed there? Or does it just restrict updates to those listed? Hopefully it's the latter.

Latter
